hpcflow/sdk/core/workflow.py

```diff
@@ -676,13 +676,15 @@ def add_task(self, task: Task, new_index=None):
         with self.batch_update():
             self._add_task(task, new_index=new_index, parent_events=[])
 
-    def add_task_after(self, task_ref):
-        # TODO: find position of new task, then call add_task
+    def add_task_after(self, task: Task, task_ref=None):
```

Comment:
Could we rename the `task` argument to `new_task` to avoid any confusion. Could you also add a type hint to `task_ref`, so the signature is like:

```def add_task_after(self, new_task: Task, task_ref: WorkflowTask = None):```

Reply:
Resolved in [e72c55b](https://github.com/hpcflow/hpcflow-new/pull/348/commits/e72c55bd53d988c530ef5f5b0a2549d947b2dc47).